templates/scripts.tmpl/cleanup.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
General piece of advice: you might want to install `shellcheck` and run it against those scripts.